server/command.go

```diff
@@ -239,6 +240,23 @@ If you see an option to create a Jira issue, you're all set! If not, refer to ou
 	return responsef(addResponseFormat, ji.GetURL(), p.GetSiteURL(), ji.GetMattermostKey(), pkey)
 }
 
+func executeAssign(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+
+	if len(args) <= 1 {
```

Comment:
Unless there can be 3 or more args, I would suggest `len(args) != 2`
Below you're joining them with spaces -- if that's intentional, you have to send over the wire with URL encoding

Reply:
That was not intentional.  using your suggestions and removing join from below.